documentation/src/main/asciidoc/topics/con_xsite_auto_offline.adoc

```diff
@@ -0,0 +1,53 @@
+[id='xsite_auto_offline-{context}']
+= Taking Sites Offline Automatically
+Backup configurations include timeout values for operations to replicate
+data to remote sites. When backup operations reach the timeout, {brandname}
+records the operation as a failure.
+
+To automatically take sites offline, you can then configure the
+number of **consecutive** failures that can occur.
+
+For example, the **NYC** backup configuration specifies five as the number of
+failures after which **NYC** goes offline. If **LON** attempts five consecutive
+backup operations that fail, {brandname} automatically takes **NYC** offline.
+**LON** then stops backing up to **NYC** until you bring the site back online.
+
+[source,xml,options="nowrap",subs=attributes+]
+----
+<backup site="NYC" strategy="ASYNC">
+	<take-offline after-failures="5"/>
+</backup>
+----
+
+You can also specify a minimum wait time for backup operations to complete
```

Comment:
this is incorrect. 
the `min-wait` is the amount of time to wait before taking the site offline. When the first failure happens, it waits this time before taking the site offline. If a successful request happens, the timer is canceled.

Reply:
fixed. thanks for clarifying this.